694.py

```diff
@@ -0,0 +1,44 @@
+class Solution:
```

Comment:
Overall this is a well written solution. This is my first time seeing a BFS solution for this classic LeetCode problem and I have learned something new today.